kubernetes/charts/edge-kubernetes/templates/_helpers.tpl

```diff
@@ -78,13 +78,22 @@ listen:
   workload_uri: "https://0.0.0.0:{{ .Values.iotedged.ports.workload }}"
 homedir: {{ .Values.iotedged.data.targetPath | quote }}
 namespace: {{ .Release.Namespace | quote }}
-proxy_image:  "{{.Values.iotedgedProxy.image.repository}}:{{.Values.iotedgedProxy.image.tag}}"
-proxy_config_path: "/etc/iotedge-proxy"
-proxy_config_map_name: "iotedged-proxy-config"
-proxy_trust_bundle_path: "/etc/trust-bundle"
-proxy_trust_bundle_config_map_name: "iotedged-proxy-trust-bundle"
-image_pull_policy: {{ .Values.iotedgedProxy.image.pullPolicy | quote }}
 device_hub_selector: ""
+proxy:
+  image: "{{.Values.iotedgedProxy.image.repository}}:{{.Values.iotedgedProxy.image.tag}}"
+  image_pull_policy: {{ .Values.iotedgedProxy.image.pullPolicy | quote }}
+  {{- if .Values.edgeAgent.registryCredentials }}
```

Comment:
We are assuming that the proxy registry==agent registry?

Reply:
nope. mistake here